server/app/server.js

```diff
@@ -53,14 +54,14 @@ const server = new FastBootAppServer({
 	afterMiddleware: (app) => {
 		app.use((err, req, res, next) => {
 			if (err) {
-				// It should never get here
-				// In theory, the Ember app handles all server errors
-				const level = levelFn(res.statusCode, err);
+				// Handle errors that don't go to FastBoot, like Bad Request etc.
+				const statusCode = Math.max(res.statusCode, err.statusCode);
+				const level = levelFn(statusCode);
 				const logFn = req.log[level].bind(req.log);
 
 				logFn(err);
 
-				res.send('Server error');
+				res.sendStatus(statusCode);
```

Comment:
are you referring to Ember or express stack trace?

`sendStatus()` sends given status code with default message:
https://expressjs.com/en/api.html#res.sendStatus